### PR TITLE
DROOLS-1375: Change count of expected coulmns in the exception message

### PR DESCRIPTION
**Pull request by jomarko**

### CODE REVIEW

File: drools-workbench-models/drools-workbench-models-guided-template/src/test/java/org/drools/workbench/models/guided/template/shared/TemplateModelTest.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
+package org.drools.workbench.models.guided.template.shared;
```
Comment (manstis) [inline drools-workbench-models/drools-workbench-models-guided-template/src/test/java/org/drools/workbench/models/guided/template/shared/TemplateModelTest.java:17]: License please!

File: drools-workbench-models/drools-workbench-models-guided-template/src/test/java/org/drools/workbench/models/guided/template/shared/TemplateModelTest.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        m.addLhsItem( p );
+
+        try {
+            m.addRow( new String[]{} );
+            Assert.fail( "Expected IlegalArgumentException" );
+        } catch ( IllegalArgumentException iae ) {
+            assertEquals( "Invalid numbers of columns: 0 expected: 1" , iae.getMessage() );
+        }
+
+
```
Comment (manstis) [inline drools-workbench-models/drools-workbench-models-guided-template/src/test/java/org/drools/workbench/models/guided/template/shared/TemplateModelTest.java]: Can you please remove the surplus white space (I wouldn't ask if you didn't need to add a license)

**Review by manstis — CHANGES_REQUESTED** [review]

**Comment by jomarko** [comment]
@manstis thanks for review. Updates applied.

**Review by manstis — APPROVED** [review]

**Comment by manstis** [comment]
Will merge when green (not expecting it to be any other colour mind you!)

Thanks @jomarko

**Pull request merged**